addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2485,6 +2485,9 @@ export class OdooEditor extends EventTarget {
             }
             ev.preventDefault();
             ev.stopPropagation();
+        } else if (ev.shiftKey && ev.key === "Enter") {
+            ev.preventDefault();
+            this._applyCommand('oShiftEnter');
```

Comment:
Is there a particular reason to use `_applyCommand` rather than `execCommand` ?

Reply:
@dmo-odoo i used this command instead of `execCommand`, to simply avoid the execution of `computeHistorySelection`
https://github.com/odoo/odoo/blob/15.0/addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js#L1317

Reply:
You mean the behavior was wrong when calling `execCommand` ?

Reply:
NO the behaviour was not wrong all we needed was a simple call to `oShiftEnter`, and i believe there was no need to call an extra method of `computedHistorySelection`

Reply:
But what if at some point we will need to restore the selection after an oShiftEnter ? It will be wrong then because `computeHistorySelection` wasn't called in that case, correct ?

Reply:
By restore selection we mean undo & redo right ? , both the cases are working even we call it from `_applyCommand` in this case, also I found similar code style [here](https://github.com/odoo/odoo/blob/15.0/addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js#L2472), also in my opinion on key down shift+enter we wont be restoring selection, what is your opinion on this?